openpype/modules/base.py

```diff
@@ -997,7 +997,17 @@ def _collect_plugin_paths(self, method_name, *args, **kwargs):
                 continue
 
             method = getattr(module, method_name)
-            paths = method(*args, **kwargs)
+            try:
+                paths = method(*args, **kwargs)
+            except Exception:
+                self.log.warning(
+                    "Failed to get plugin paths from module {}.".format(
+                        module.__class__.__name__
+                    ),
+                    exc_info=True
+                )
+                continue
```

Comment:
Should we include the method name so we know which of the plugins path retrievals it failed on?